panel/lab/components/filepreviews/2_image/index.vue

```diff
@@ -1,61 +1,52 @@
 <template>
 	<k-lab-examples>
 		<k-lab-example label="Landscape">
-			<k-file-preview
-				:focus="focus"
-				:focusable="true"
+			<k-image-file-preview
 				:details="details"
 				:image="image"
 				url="https://getkirby.com"
-				@focus="focus = $event"
+				class="k-file-preview"
```

Comment:
Wouldn't this class be something that the file preview components should already have applied?

Reply:
Ah, sorry. I saw further down in the code that it is applied if you use the k-file-preview wrapper

Reply:
Yes, I was myself torn gif this is the best way, but couldn't make up my mind.